<?xml version="1.0"?>
<launch>
    <node pkg="movo_object_search" type="object_search_main.py"  name="object_search_main" output="screen" >
        <param name="num_regions" value="1"/>
        <param name="world_x_res" value="3"/>
        <param name="world_y_res" value="3"/>
        <param name="world_z_res" value="2"/>
        <param name="horizontal_cell_size" value="1.0"/>
        <param name="vertical_cell_size" value="0.2"/>
        <param name="trans_vel" value="0.3"/>
        <param name="ang_vel" value="30"/>
        <param name="goal_pose_tolerance" value="0.05"/>
        <param name="goal_angle_tolerance" value="0.3"/>
    </node>
</launch>
